ACLWeb.js

```diff
@@ -86,7 +88,7 @@ function doWeb(doc, url) {
 		let paperURL = url.replace('.bib', '');
 		ZU.processDocuments(paperURL, scrape);
 	}
-	else if (doc.contentType === 'application/pdf') {
+	else if (doc.contentType === 'application/pdf' || url.endsWith('.pdf')) {
```

Comment:
Are both checks here needed?

Reply:
I'm not sure... @GuyAglionby mentioned something about a `contentType` bug in Firefox that I figure this might fix once further URL changes are made on the site.